railties/lib/rails/generators/rails/app/templates/config/databases/mysql.yml.tt

```diff
@@ -56,3 +56,6 @@ production:
   database: <%= app_name %>_production
   username: <%= app_name %>
   password: <%%= ENV['<%= app_name.upcase %>_DATABASE_PASSWORD'] %>
+# To enforce TLS connection to your backend, comment out the following two lines
+#  sslmode: 'REQUIRED'
+#  sslca: config/certificateyoutrust.pem
```

Comment:
Love seeing TLS strongly encouraged. This glosses over the necessary work a good bit. Figuring out which CA cert you should be using and putting it in place is the hard part—more than a line to uncomment. Perhaps Rails Guides are a place to offer this kind of guidance? We could link to that in the commentary earlier in this file.

Reply:
Sounds great. I think linking to rails guides and provide example of certificates (maybe AWS and few others) would help.